lib/AST/UnqualifiedLookup.cpp

```diff
@@ -708,16 +705,10 @@ void UnqualifiedLookupFactory::printScopes(raw_ostream &out) const {
 
 void UnqualifiedLookupFactory::printResults(raw_ostream &out) const {
   for (auto i : indices(Results)) {
-    if (i == resultsSizeBeforeLocalsPass)
-      out << "============== next pass ============\n";
     out << i << ": ";
     Results[i].print(out);
     out << "\n";
   }
-  if (resultsSizeBeforeLocalsPass == Results.size())
-    out << "============== next pass ============\n";
-  if (resultsSizeBeforeLocalsPass == ~0u)
-    out << "never tried locals\n\n";
```

Comment:
This and the block at 712 were useful for my debugging. But if you don't need them, it's fine by me.

Reply:
I deleted it because `resultsSizeBeforeLocalsPass` was never written to, so reading from it always produced the value `~0U`. It appears that the only usage of this variable was in the DeclContext-based unqualified lookup that I deleted.

Reply:
Good move!